netci.groovy

```diff
@@ -332,6 +332,51 @@ def supportedFullCycleInnerloopPlatforms = ['Windows_NT', 'Ubuntu14.04', 'Ubuntu
     }
 }
 
+// **************************
+// Define outerloop testing on Windows_NT for UAP, run locally on each machine.
+// **************************
+
+[true, false].each { isPR -> 
+    configurationGroupList.each { configurationGroup ->
+        def os = 'Windows_NT'
+        def newJobName = "outerloop_selfhost_${os.toLowerCase()}_${configurationGroup.toLowerCase()}"
+        def newJob = job(Utilities.getFullJobName(project, newJobName, isPR))
+        def targetGroup = "uap"
+        
+        newJob.with {
+            steps {
+                batchFile("build.cmd -framework:${targetGroup} -${configurationGroup} -os:${osGroupMap[os]}")
+                batchFile("build-tests.cmd -framework:${targetGroup} -${configurationGroup} -os:${osGroupMap[os]} -outerloop -- /p:ServiceUri=%WcfServiceUri% /p:SSL_Available=true /p:Root_Certificate_Installed=true /p:Client_Certificate_Installed=true /p:Peer_Certificate_Installed=true /p:IsCIBuild=true")
```

Comment:
This is called outerloop_selfhost but you are specifying ServiceUri. I suspect one of these is wrong.